packages/fx-core/src/core/middleware/projectVersionChecker.ts

```diff
@@ -46,7 +46,7 @@ async function showDialog(ctx: CoreHookContext) {
   if (inputs.platform === Platform.VSCode) {
     await TOOLS?.ui.showMessage(
       "warn",
-      getLocalizedString("core.projectVersionChecker.vscodeUseNewVersion"),
+      getLocalizedString("core.projectVersionChecker.vscodeUseNewVersion", "Teams Toolkit 5.0.0"),
```

Comment:
It's may be hard to determine which version to install. For example, maybe one day Teams Toolkit 6.0 bumps the version in teamsapp.yml to 2.0.0, then the hint "Teams Toolkit 5.0.0" would be incorrect when user use old Teams Toolkit.

Should we just ask users install latest toolkit and provide an aka link here?

Reply:
The message has been discussed with @jidddddd.
Actually, the message just suggests a lowest toolkit version
![image](https://user-images.githubusercontent.com/16380704/212716768-bd221e56-d3ba-4e6b-8919-d47cfbb4c92f.png)